src/util/models.py

```diff
@@ -76,6 +76,38 @@ def remove_model(cr, model, drop_table=True, ignore_m2m=()):
 
     # remove references
     for ir in indirect_references(cr):
+        if ir.company_dependent_comodel:
+            if ir.company_dependent_comodel != 'ir.model':
+                # we don't handle company dependent here
+                # the field should be field_remove when upgrade ir.company_dependent_comodel
+                # since the referenced model here is removed
+                continue
+            # clean on delete set null
+            cr.execute("SELECT id FROM ir_model WHERE name = %s", [model])
+            [mod_id] = cr.fetchone() or [None]
+            if mod_id:
+                cr.exeucte(
```

Comment:
like remove_record, I think the below code shouldn't be executed since I cannot image the usage of
`model_id = fields.Many2one('ir.model', company_dependent=True)`
Should we remove it and just raise a warning?

Reply:
`exeucte` -> `execute`

> I think the below code shouldn't be executed since I cannot image the usage of...

If it was allowed, we would get a case most probably. Thus I'd suggest you keep it.